packages/service-clients/end-to-end-tests/azure-client/src/test/gc.spec.ts

```diff
@@ -0,0 +1,142 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+
+import { AzureClient } from "@fluidframework/azure-client";
+import { ConnectionState } from "@fluidframework/container-loader";
+import {
+	ContainerSchema,
+	type CompatibilityMode,
+	type IFluidContainer,
+} from "@fluidframework/fluid-static";
+import { SharedMap } from "@fluidframework/map/internal";
+import { MockLogger } from "@fluidframework/telemetry-utils/internal";
+import { timeoutPromise } from "@fluidframework/test-utils/internal";
+import type { AxiosResponse } from "axios";
+
+// eslint-disable-next-line import/no-internal-modules
+import type { IGarbageCollectorConfigs } from "../../../../../runtime/container-runtime/lib/gc/gcDefinitions.js";
+
+import {
+	createAzureClient,
+	createContainerFromPayload,
+	getContainerIdFromPayloadResponse,
+} from "./AzureClientFactory.js";
+import * as ephemeralSummaryTrees from "./ephemeralSummaryTrees.js";
+import { getTestMatrix } from "./utils.js";
+
+const testMatrix = getTestMatrix();
+for (const testOpts of testMatrix) {
+	describe.only(`Garbage collection with AzureClient (${testOpts.variant})`, () => {
+		const connectTimeoutMs = 10_000;
+		const isEphemeral: boolean = testOpts.options.isEphemeral;
+		let client: AzureClient;
+		let mockLogger: MockLogger;
+		const schema = {
+			initialObjects: {
+				map1: SharedMap,
+			},
+		} satisfies ContainerSchema;
+
+		beforeEach("createAzureClient", () => {
+			mockLogger = new MockLogger();
+			client = createAzureClient(undefined /* id */, undefined /* name */, mockLogger);
+		});
+
+		async function createFluidContainer(
+			compatibilityMode: CompatibilityMode,
+		): Promise<IFluidContainer> {
+			if (isEphemeral) {
+				const containerResponse: AxiosResponse | undefined = await createContainerFromPayload(
+					ephemeralSummaryTrees.createContainerWithSharedTree,
```

Comment:
This seems like it probably doesn't load correctly, since the created container uses SharedTree but the `schema` only specifies SharedMap?  I'd guess the container from getContainer below probably errors/closes but this doesn't cause the test to fail.

setDDSesAsInitialObjectsForContainer (tree1) might be a reasonable one to reuse (from ddsTests.spec.ts)

Reply:
Ah, it's leftover because I copied the test from SharedTree. I will fix it.